app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,35 @@
+class AccountActivationsController < ApplicationController
+  before_action :get_user, only: :edit
+
+  def edit
```

Comment:
```suggestion
  # GET /account_activations/:id
  def edit
```
Thêm comment routes trên action của controller để tường minh hơn